src/fsharp/vs/service.fs

```diff
@@ -1473,12 +1473,10 @@ type TypeCheckInfo
                // custom builders, custom operations get colored as keywords
                | CNR(_, (Item.CustomBuilder _ | Item.CustomOperation _), ItemOccurence.Use, _, _, _, m) -> 
                    yield (m, FSharpTokenColorKind.Keyword) 
-#if COLORIZE_TYPES
```

Comment:
I'm concerned that the type colorization logic may be pretty buggy or at least untested. Please compare with the Visual F# Power Tools paths, which use the FCS Symbols API to do colorization rather than this code (which is old and not exercised by any clients at the moment - one of the reasons we added the Symbols API was to be able to do this outside FCS rather than inside)